app/redux/products.js

```diff
@@ -5,7 +5,7 @@ const SET_PRODUCTS = 'SET_PRODUCTS';
 const setProducts = products => {
   return {
     type: SET_PRODUCTS,
-    products
+    products:products
```

Comment:
FYI you can just write `products` as this will destructure it into `products: products`